dotenv-example

```diff
@@ -0,0 +1,9 @@
+# general logconf setting
+LOGCONF=debug-logconf.yml
+
+# general repo setting
+GDB_REPO="lwua23"
+
+# scheduler settings
+SCHEDULER_PERIOD=5 # seconds
```

Comment:
more context in env name useful?
`INGEST_SCHEDULE_PERIOD` and `INGEST_SCHEDULE_WAIT` ?